.github/workflows/actions.yml

```diff
@@ -29,12 +29,19 @@ jobs:
           mamba-version: "*"
           environment-file: environment.yml
 
+      - name: Generate cache key for submodule
+        id: generate-cache-key
+        run: |
+          # Get the submodule commit SHA
+          SUBMODULE_SHA=$(git submodule status | awk '{print $1}' | sha256sum | cut -d ' ' -f 1)
+          echo "SUBMODULE_KEY=$SUBMODULE_SHA" >> $GITHUB_ENV
```

Comment:
Is this line correct? I have a message
bash: $GITHUB_ENV: ambiguous redirect 
if I am trying to execute it locally.